src/interaction/inspector.jl

```diff
@@ -426,14 +416,10 @@ function update_tooltip_alignment!(inspector, proj_pos)
     return
 end
 
-
-
 ################################################################################
 ### show_data for primitive plots
 ################################################################################
 
-
-
```

Comment:
Do we want larger spacings like this to get removed? I find them quite helpful to make sections easy to notice as you scroll by.

Reply:
That's the effect of `remove_extra_newlines = true` and it can be disabled.